packages/react/src/components/Pivot/Pivot.base.tsx

```diff
@@ -244,9 +244,10 @@ export const PivotBase: React.FunctionComponent<IPivotProps> = React.forwardRef<
       <div role="toolbar" {...divProps} ref={ref}>
         <FocusZone
           componentRef={focusZoneRef}
-          direction={FocusZoneDirection.horizontal}
-          className={classNames.root}
+          className={css(classNames.root, focusZoneProps.className)}
```

Comment:
Thanks for this change! However, you need to move this after the spread in line 250 because if not `{...focusZoneProps}` will overwrite this value.